kubetest/kops.go

```diff
@@ -173,6 +173,18 @@ func (k kops) isGoogleCloud() bool {
 }
 
 func (k kops) Up() error {
+	// If we downloaded kubernetes, pass that version to kops
+	if k.kubeVersion == "" {
+		kubeReleaseUrl := os.Getenv("KUBERNETES_RELEASE_URL")
+		kubeRelease := os.Getenv("KUBERNETES_RELEASE")
+		if kubeReleaseUrl != "" && kubeRelease != "" {
```

Comment:
This is pretty obviously not a great way to get the URL.  Suggestions welcome!  (should kubetest/extract.go set it somewhere?)

Reply:
assume you want to set `k.kubeVersion` probably when creating the kops deployer

Reply:
I had a look at that, but...

* `getDeployer` (which builds the kops object) is called before `acquireKubernetes`.  We could swap the calls, obviously.
* `acquireKubernetes` has a few strategies for getting k8s, but doesn't return the URL cleanly where it is found AFAICT.  It does set the env variables as part of `getKube`.

I guess the best option is for `acquireKubernetes` to return the location(s) where k8s is found.  And/or for the deployer to call `acquireKubernetes` - because, for example, kops doesn't need all the artifacts locally.

I think a real fix here is a pretty invasive refactor...

Reply:
maybe add a TODO if this is blocking the kops-gce suite